app/src/main/java/org/p2p/wallet/settings/ui/settings/presenter/SettingsItem.kt

```diff
@@ -25,7 +25,7 @@ sealed interface SettingsItem {
     ) : SettingsItem
 
     data class ComplexSettingsItem(
-        @StringRes val settingNameRes: Int,
+        @StringRes val nameRes: Int,
```

Comment:
why not settingsName? don't you think that name here is too abstract? is it item name or user name? i would rather be more specific here, it's always not redundant

Reply:
Because we already have the class name.
Settings item has settings id seems redundant for sure.
And it's the name of the settings, not the settings name